LPS-91942

dgarciasarai opened the pull request.

wcao20170619 committed:
LPS-84659 portal-search-web: Sort Portlet

https://issues.liferay.com/browse/LPS-84659

Co-authored-by: weicao <wade.cao@liferay.com>
Co-authored-by: André de Oliveira <andre.oliveira@liferay.com>

wcao20170619 committed:
LPS-84659 portal-search-api: Sort Builder

https://issues.liferay.com/browse/LPS-84659

Co-authored-by: weicao <wade.cao@liferay.com>
Co-authored-by: André de Oliveira <andre.oliveira@liferay.com>

arboliveira committed:
LPS-84659 portal-search: Sort Builder

https://issues.liferay.com/browse/LPS-84659

Co-authored-by: weicao <wade.cao@liferay.com>
Co-authored-by: André de Oliveira <andre.oliveira@liferay.com>

jbalsas committed:
LPS-91967 Removes lock files

ChrisKian committed:
LPS-91588 No need to check against User table since screenname is added to a new group during User creation process

mtambara committed:
LPS-92022 Have server trigger test events and notify client

jbalsas committed:
LPS-91967 Adds general modules/package.json file with the workspaces definitions

jbalsas committed:
LPS-91967 Adds common yarn executable

jbalsas committed:
LPS-91967 Updates npm dependencies

jbalsas committed:
LPS-91967 Updates build.gradle files to remove unnecessar jsCompile dependencies

jbalsas committed:
LPS-91967 Updates build dependencies

jbalsas committed:
LPS-91967 Removes unnecessary .liferaynpmscriptsrc files (improved defaults)

jbalsas committed:
LPS-91967 Updates yarn.lock file

petershin committed:
LPS-91967 Update Ant task compile (run setUpYarn)

petershin committed:
LPS-91967 Configure node plugin for Yarn

mtambara committed:
LPS-92022 Move all test pre-processing and execution to the test bundle

patriciajeanperez committed:
LRQA-47415 Go to list view

mtambara committed:
LPS-92022 Remove JMXTestRunner

mtambara committed:
LPS-92022 Move client side test class initialization to before bundle install

mtambara committed:
LPS-92022 Merge JMXTestRunner into ArquillianBundleActivator

mtambara committed:
LPS-92022 Not thrown

petershin committed:
LPS-91967 Add SF suppressions for yarn-1.13.0.js

petershin committed:
LPS-91967 Configure JS plugins for Yarn

mtambara committed:
LPS-92022 Execute test in BundleListener to ensure components provided in test bundles can start first

arboliveira committed:
LPS-84659 portal-search-elasticsearch6-impl: Tolerate null sort order

https://issues.liferay.com/browse/LPS-84659

Co-authored-by: weicao <wade.cao@liferay.com>
Co-authored-by: André de Oliveira <andre.oliveira@liferay.com>

Commit:
LPS-90056 - Refactor and rename

mtambara committed:
LPS-92022 Connect on random socket so tests don't interfere with each other

mtambara committed:
LPS-92022 use the same socket fo the entire test

mtambara committed:
LPS-92022 Collect and fire all framework failures at the end of the test

arboliveira committed:
LPS-84659 portal-search: Tolerate null sort order

https://issues.liferay.com/browse/LPS-84659

Co-authored-by: weicao <wade.cao@liferay.com>
Co-authored-by: André de Oliveira <andre.oliveira@liferay.com>

pavel-savinov committed:
LPS-92236 MVC resource command to get particular asset field value

sandrodw3 committed:
LPS-92236 URL for getting an asset field value added to state and editables component

p2kmgcl committed:
LPS-92240 Add FragmentsEditorShim

p2kmgcl committed:
LPS-92240 Add ColumnOverlayGrid

p2kmgcl committed:
LPS-92240 Allow configuring padding

p2kmgcl committed:
LPS-92240 Allow configuring containerType

Commit:
LPS-90056 - Enable fragment editor via reducer

p2kmgcl committed:
LPS-92240 Add highlightedColumn property

jbalsas committed:
LPS-91967 Updates theme builds to use local styled and unstyled themes

sandrodw3 committed:
LPS-92236 Added property to state for storing mapped asset field value

Commit:
LPS-90056 - Add select image button to image properties panel

Commit:
LPS-90056 - SF

sandrodw3 committed:
LPS-92236 Added function to check if a field is mapped to an asset

sandrodw3 committed:
LPS-92236 Added method for updating mapped field value

sandrodw3 committed:
LPS-92236 Updated mapped field value when editableValues or URL change

Commit:
LPS-90056 - Add clear image button to image properties panel

Commit:
LPS-90056 - SF

sandrodw3 committed:
LPS-92240 SF

yichenroy committed:
LPS-91967 Make sure to setup-yarn in prepare-test-environment

Commit:
LRQA-47408 Move the locators to appropriate path file

Commit:
LRQA-47408 Add a macro to go to service provider

Commit:
LRQA-47408 Add a macro to configure service provider

Commit:
LRQA-47408 Add a case to assert SSO with sign authn requests

p2kmgcl committed:
LPS-92240 Sort SCSS variables

p2kmgcl committed:
LPS-92236 Rename _isMappedToAsset to _isMappedToAssetEntry

sandrodw3 committed:
LPS-92241 Modified cursor from move to grab for drag handlers

p2kmgcl committed:
LPS-90056 Add missing jsdocs

jkappler committed:
LPS-92236 Make return value consistent

p2kmgcl committed:
LPS-92236 Prevent extra renders

sandrodw3 committed:
LPS-92241 Added template for column resizer

sandrodw3 committed:
LPS-92241 Added styles for column resizer

p2kmgcl committed:
LPS-90056 Update imagePropertiesPanel for asset entry mapping

p2kmgcl committed:
LPS-91982 Move padding options to constants

sandrodw3 committed:
LPS-92241 Column resizer visible property added to state on prepareStateForRender

sandrodw3 committed:
LPS-92241 Added resizer to columns

petershin committed:
LPS-91967 Move existing logic to new task yarnInstall

petershin committed:
LPS-91967 Cache node_modules directory

shuyangzhou committed:
LPS-92022 We can retry on binding, but we should not retry on ServerSocketChannel opening failure, especially this would leave unclosed ServerSocketChannel on binding failure.

shuyangzhou committed:
LPS-92022 This should not be a random retry, but a sequential searching for an available port.

shuyangzhou committed:
LPS-92022 loopback address is good enough for now. Once we solve the remote bundle installing issue, we will need to revisit here to make it binding to all localhost address in order to be open for remote connecting.

shuyangzhou committed:
LPS-92022 Inline

shuyangzhou committed:
LPS-92022 Stop mutating a static field, that is generally a bad thing to do. Later on (if necessary) we can make this start port value configurable.

shuyangzhou committed:
LPS-92022 This can be static.

shuyangzhou committed:
LPS-92022 Thread should not start until test bundle is installed.

shuyangzhou committed:
LPS-92022 Considering the server connects back for exactly once for ServerSocket lifespan, leave the ServerSocket and Socket lifecycle control to the top run() method. The thread only incharges of pumping data back.

shuyangzhou committed:
LPS-92022 SF

shuyangzhou committed:
LPS-92022 We know the main test execution thread is not non-daemon. Due to we want to join the data pumpping thread back, it is pointless to make it daemon.

shuyangzhou committed:
LPS-92022 Properly handle exception suppression

shuyangzhou committed:
LPS-92022 No need to buffer anything, socket is backed with buffer already. If we not happy with the size, we should configure the socket buffer, rather than buffer it again.

shuyangzhou committed:
LPS-92022 Lower access level

shuyangzhou committed:
LPS-92022 Pull all bundle header names into a constants class to avoid typos. And namespace them all to avoid future collisions.

shuyangzhou committed:
LPS-92022 Simplify

shuyangzhou committed:
LPS-92022 We are registering the listener under our own BundleContext. When Bundle stops, it automatically takes care of all the unregister under its BundleContext, so we don't have to do this explicitly.

shuyangzhou committed:
LPS-92022 SF

shuyangzhou committed:
LPS-92022 Use InetSocketAddress

shuyangzhou committed:
LPS-92022 Load directly from Bundle

shuyangzhou committed:
LPS-92022 SF

jbalsas committed:
LPS-91967 Fixes analytics-client-web tests

shuyangzhou committed:
LPS-92022 Move socket lifecycle control out of ClientBridge

shuyangzhou committed:
LPS-92022 This is a weid loop...

shuyangzhou committed:
LPS-92022 Explicitly asking for Serializable

shuyangzhou committed:
LPS-92022 Remove ClientBridge

shuyangzhou committed:
LPS-92022 Use EOF as kill

shuyangzhou committed:
LPS-92022 Let's be strict, abort on the first framework failure, as most likely is not recoverable. Keep running could lead to OOM.

shuyangzhou committed:
LPS-92022 Create RunNotifierCommand to avoid doing string based protocol (which is not type safe).

dgarciasarai committed:
LPS-91942 Render fields inside collapse

Commit:
LPS-91995 Fixing test case

shuyangzhou committed:
LPS-92022 Rethrow rather than just break. The test may have more than 1 @Inject usage. A simple break is enough to break out the upper level loop, which would lead to a hanging on next wait.

shuyangzhou committed:
LPS-92022 Bundle starting can fail, in case of that we still need to ensure the bundle being uninstalled.

shuyangzhou committed:
LPS-92022 No need to start a new thread on the client side.

shuyangzhou committed:
LPS-92022 SF

shuyangzhou committed:
LPS-92022 Duplicated catching

shuyangzhou committed:
LPS-92022 Refine the server execution strategy. Run test in a dedicated thread to have it isolated.temp

brianchandotcom committed:
LPS-92511 Remove SafeConsumer, so ugly

brianchandotcom committed:
LPS-92511 apply

timpak committed:
LRQA-47556 Remove unused locator

brianchiu committed:
LRQA-47622 'Publish Page' button changed into 'Publish'

brianchiu committed:
LRQA-47622 Update autosave text

ruben-pulido committed:
LPS-92531 Add assetListEntryKey to AssetList Entity

ruben-pulido committed:
LPS-92531 Add finder for AssetListEntryKey

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

4lejandrito committed:
LPS-92530 Adding menu to the detail view

ruben-pulido committed:
LPS-92531 Save asset list entry key as in FragmentEntryKey

4lejandrito committed:
LPS-92530 Not needed

The menu won't show when it has no options

4lejandrito committed:
LPS-92530 Adding navigation to the list view

4lejandrito committed:
LPS-92530 SF

dgarciasarai committed:
LPS-91942 SF

boton committed:
LPS-92530 Add the right class to file context actions dropdown trigger

adolfopa committed:
LPS-92208 Add all mime types we support

Despite its name, the PDFProcessor supports a lot of document
formats.  Add all the ones that we want to have previews for.

adolfopa committed:
LPS-92208 Rename, no longer specific to PDFs

pavel-savinov committed:
LPS-92539 Remove home page badge

pavel-savinov committed:
LPS-92539 Adjust discard draft buttons styles

pavel-savinov committed:
LPS-92539 Draft state depends on the publish date

jkappler committed:
LPS-92446 Copy related classes from asset display module to info module changing asset > info. Take advantage and remove methods that will be used in helper class in the future

jkappler committed:
LPS-92446 Create tracker for contributed infoDisplayContributor fields

andresfulla commented:
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

inacionery committed:
LPS-83472 Add missing parameter in template url

jkappler committed:
LPS-92531 Adds integration test

interaminense committed:
LPS-92563 Update version of webpack-cli package

nikki-pru committed:
LRQA-47651 Make path usable for both CP and PG assertions

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/analytics/.gitrepo'.

brianchandotcom committed:
LPS-83748 SF

brianchandotcom committed:
LPS-91995 better name

brianchandotcom committed:
LPS-91995 only 1

brianchandotcom committed:
LPS-91995 auto SF

brianchandotcom committed:
LPS-83472 match modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/edit_template.jsp

joshchong committed:
LPS-92097 Update version locator for functional test

jkappler committed:
LPS-92531 Auto

brianchandotcom committed:
Record reference to liferay-portal/master-private with git-commit-portal-private as 9dd3a3b5a9f4ae07e2022c87eca330eaa6d348e0

xbrianlee committed:
LRQA-47408 SF

brianchandotcom committed:
LPS-91549 allow *-client

brianchandotcom committed:
LPS-91549 prep next

brianchandotcom committed:
LPS-91549 prep next

Commit:
LPS-91799 Create permission resources for data definition

Commit:
LPS-91799 Create internal model for permission check

Commit:
LPS-91799 Create permission classes to InternalDataDefinition

Commit:
LPS-91799 Change service calls to use new data definition model class

joshchong committed:
LRQA-47655 Use function that doesn't rely on element present

joshchong committed:
LRQA-47655 SF

Commit:
LPS-91799 check permissons before perform some operations

brianchandotcom committed:
LPS-91799 as used

brianchandotcom committed:
LPS-91799 Less chance of me screwing it up

brianchandotcom committed:
LPS-91799 Simplify

brianchandotcom committed:
LPS-91799 Call it what it is

brianchandotcom committed:
LPS-91799 SF

BryanEngler committed:
LPS-92582 portal-search-web: Prevent ambiguous reference

https://issues.liferay.com/browse/LPS-92582

timpak committed:
LRQA-47556 Use LexiconCard macro instead

brianchiu committed:
LPS-92461 Remove iframe selection

timpak committed:
LRQA-47556 SF

Commit:
LPS-92181 - Updating Alloy 3.1.x to c54aea0ae4ed088e52f98fd4c08e19241a09fc3e

petershin committed:
LPS-91511 Check if the method has path parameters

pavel-savinov committed:
LPS-92585 Escape portlet resource to prevent XSS in asset publisher

brianchandotcom committed:
Record reference to liferay-portal/master-private with git-commit-portal-private as a05eac934e1873dd2b3f0ce3ae94a6a94f4408f3

pavel-savinov committed:
LPS-91733 Include theme-specific CSS only in case when current scope group is actually using the theme

pavel-savinov committed:
LPS-91724 Use best align feature to ensure that floating toolbar is not hidden

pavel-savinov committed:
LPS-92566 Default language id must reflect site default language, not current language of the user

pavel-savinov committed:
LPS-91717 Prevent sections from being added to columns

pavel-savinov committed:
LPS-91201 Pointer cursor for the page type selector

brianchandotcom committed:
LPS-91733 SF

brianchandotcom committed:
Record reference to liferay-portal/master-private with git-commit-portal-private as e9b673214195f9520ac54c221b2f6527bd6915e1

javiergamarra committed:
LPS-92551 add primary, contact information and keywords

javiergamarra committed:
LPS-92551 generate

javiergamarra committed:
LPS-92551 extract util class to reuse it in organizations

javiergamarra committed:
LPS-92551 embed contact information

javiergamarra committed:
LPS-92551 add keywords

pavel-savinov committed:
LPS-92522 Avoid NPE in case when there is no className in the fragment editable values

pavel-savinov committed:
LPS-92522 Exclude segment experiences creation from staging and/org export-import process since there is no staging support for segments yet

shuyangzhou committed:
LPS-92022 Add pass code support to prevent random client from connecting.

shuyangzhou committed:
LPS-92022 Invoke test in the real junit way, classblock -> methodblock, and rule/classrule at proper level. No more arquillian classrule, beforeclass/afterclass hacks.

# Conflicts:
#	modules/test/arquillian-extension-junit-bridge/src/main/java/com/liferay/arquillian/extension/junit/bridge/server/TestExecutorRunnable.java

javiergamarra committed:
LPS-89732 change to writeOnly and remove put because logic is problematic

javiergamarra committed:
LPS-89732 regenerate

javiergamarra committed:
LPS-89732 adapt

javiergamarra committed:
LPS-89739 remove structuredContentImage and add descriptions

javiergamarra committed:
LPS-89739 regenerate

javiergamarra committed:
LPS-89739 add document and extract util

javiergamarra committed:
LPS-92462 add child information

javiergamarra committed:
LPS-92462 don't add the schema if there is already a reference

jbalsas committed:
LPS-91967 Updates lock file to latest master changes

jbalsas committed:
LPS-91967 Adds new document-library-preview-document module as a workspace

brianchandotcom committed:
LPS-92022 SF

brianchandotcom committed:
LPS-92022 I assume order matters here

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/apps/analytics/.gitrepo'.

brianchandotcom committed:
LPS-89739 Sort

brianchandotcom committed:
LPS-91967 Expose this?

brianchandotcom committed:
LPS-91967 Sort

hhuijser committed:
LPS-91549 Update documentation

arboliveira committed:
LPS-84659 portal-search-test: Sort Builder

https://issues.liferay.com/browse/LPS-84659

Co-authored-by: weicao <wade.cao@liferay.com>
Co-authored-by: André de Oliveira <andre.oliveira@liferay.com>

brianchandotcom committed:
LPS-92462 SF

brianchandotcom committed:
LPS-91549 toJSON for parser

hhuijser committed:
LPS-84119 Catch incorrect line break

hhuijser committed:
LPS-84119 Sort

brianchandotcom committed:
LPS-91549 regen

hhuijser committed:
LPS-84119 Auto SF

hhuijser committed:
LPS-91420 Add logic to sort annotations and annotationMemberValuePairs

hhuijser committed:
LPS-91420 Skip

brianchandotcom committed:
LPS-91549 Rename to SerDes

brianchandotcom committed:
LPS-91549 regen

brianchandotcom committed:
LPS-91549 add BaseJSONParser

brianchandotcom committed:
LPS-91549 regen

brianchandotcom committed:
LPS-91549 carbon copy from https://github.com/brianchandotcom/liferay-portal/blob/6090bda987d35901bcfb3a532920d2d23c3dc57a/modules/apps/headless/headless-foundation/headless-foundation-api/src/main/java/com/liferay/headless/foundation/dto/v1_0/parsing/JSONParser.java from https://github.com/brianchandotcom/liferay-portal/pull/69225/commits

brianchandotcom committed:
LPS-91549 get rid of this for now

brianchandotcom committed:
LPS-91549 Rename

brianchandotcom committed:
LPS-91549 Private whatever can be privated

brianchandotcom committed:
LPS-91549 Rename

brianchandotcom committed:
LPS-91549 Fix imports

brianchandotcom committed:
LPS-91549 Fix T

brianchandotcom committed:
LPS-91549 Minor changes

brianchandotcom committed:
LPS-91549 Should not be static

brianchandotcom committed:
LPS-91549 private it

brianchandotcom committed:
LPS-91549 minor changes

brianchandotcom committed:
LPS-91549 minor changes

brianchandotcom committed:
LPS-91549 Sort it somehow

brianchandotcom committed:
LPS-91549 auto SF

brianchandotcom committed:
LPS-91549 rename

brianchandotcom committed:
LPS-91549 Wordsmith

brianchandotcom committed:
LPS-91549 Wordsmith

brianchandotcom committed:
LPS-91549 Wordsmith

brianchandotcom committed:
LPS-91549 Wordsmith

brianchandotcom committed:
LPS-91549 SF

brianchandotcom committed:
LPS-91549 SF

brianchandotcom committed:
LPS-91549 Rename

brianchandotcom committed:
LPS-91549 Sort

brianchandotcom committed:
LPS-91549 SF

brianchandotcom committed:
LPS-91549 Wordsmith

brianchandotcom committed:
LPS-91549 SF

brianchandotcom committed:
LPS-91549 Rename

brianchandotcom committed:
LPS-91549 Should not be static

brianchandotcom committed:
LPS-91549 Sort

brianchandotcom committed:
LPS-91549 Incorporate

brianchandotcom committed:
LPS-91549 Should not have any dependencies

brianchandotcom committed:
LPS-91549 Regen

brianchandotcom committed:
LPS-91549 Add clients

brianchandotcom committed:
LPS-91549 prevent conflict (Foundation Email DTO has a field called email)

brianchandotcom committed:
LPS-84659 portlet is internal

brianchandotcom committed:
LPS-84659 SF

pavel-savinov committed:
Revert "LPS-92181 - Updating Alloy 3.1.x to c54aea0ae4ed088e52f98fd4c08e19241a09fc3e"

This reverts commit 8dba1a31

pavel-savinov committed:
Revert "LPS-91367 - Updating Alloy 3.1.x to 25b0e99ce5db7d3e0101e27f329ff104f5101423"

This reverts commit 2f7427e0

dgarciasarai committed:
LPS-91942 Fix drag and drop properties

dgarciasarai committed:
LPS-91942 Sidebar title

dgarciasarai committed:
LPS-91942 buildLang

dgarciasarai committed:
LPS-91942 Collapse style

dgarciasarai committed:
LPS-91942 Animate collapse icon

andresfulla commented:
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

andresfulla commented on modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_sidebar/CriteriaSidebarCollapse.es.js:
```diff
@@ -85,7 +85,7 @@ class CriteriaSidebarCollapse extends Component {
 									{propertyGroup.name}
 									<span className="collapse-icon">
-										<ClayIcon iconName={`angle-${propertyGroup.propertyKey === propertyKey ? 'down' : 'right'}`} />
+										<ClayIcon iconName="angle-right" className={propertyGroup.propertyKey === propertyKey ? 'active' : ''} />
```
```javascript
const targetClasses = getCN(
	{
         	'active': (propertyGroup.propertyKey === propertyKey)
	}
);
```

andresfulla commented on modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_sidebar/CriteriaSidebarCollapse.es.js:
```diff
@@ -77,13 +77,17 @@ class CriteriaSidebarCollapse extends Component {
-							<div className="sidebar-header" onClick={this._handleClick(propertyGroup.propertyKey, propertyGroup.propertyKey === propertyKey)}>
-								{propertyGroup.name}
-								<ClayIcon iconName="angle-down" />
+						return (<li className={`sidebar-collapse-${propertyGroup.propertyKey}`} key={propertyGroup.propertyKey}>
```
getCN

andresfulla commented on modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_builder/ContributorBuilder.es.js:
```diff
@@ -120,10 +120,10 @@ class ContributorBuilder extends React.Component {
 				diffState = {
 					contributors: prevState.contributors.map(
-						(contributor, i) => {
+						contributor => {
```
💃

andresfulla commented:
Pull request submitted to https://github.com/dgarciasarai/liferay-portal/pull/34. [See changes here.](https://github.com/andresfulla/liferay-portal/compare/be191d369952~1...be191d369952) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*
